lib/govuk_sidekiq/error_handling.rb

```diff
@@ -0,0 +1,33 @@
+module GovukSidekiq
+  module ErrorHandling
+    class RetryWorker < StandardException
```

Comment:
Should this be StandardError?

Reply:
yes it should! I'd accidentally left out `_spec` at the end of the tests file so they weren't even running...